src/start.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+clear
```

Comment:
Oj, niefajnie, że zawartość konsoli znika :)

Reply:
Wydawało mi się że to właśnie będzie fajne :)